test/goldens/end2end/__timestamp.txt

```diff
@@ -1 +1 @@
-2022-10-20T07:30:43.697198Z
+2023-05-12T12:55:25.249916Z
```

Comment:
Should we move this to a separate PR?

Reply:
We want these changes either in this PR or the very next one, because the old version set of the packages and their dependencies doesn't make much sense now with 3.0 SDK.